src/app/copy/block-copy/example/block-copy-example.component.ts

```diff
@@ -0,0 +1,55 @@
+import {
```

Comment:
I'd like to split the examples into separate tabs -- see my comment for inline-copy-example